flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
this is disappointing.

Reply:
It is my first draft of flatpak building code. In general I have no experience if it is working for other architectures.

I need to discuss with NEST developer (https://github.com/nest/nest-simulator).

Reply:
isn't idea being Electron that it is all javascript? The arch shouldn't matter.

Reply:
That is correct, the frontend is all javascript. But the backend is `nest-simulator` running in cpp and it provides python interface. I asked NEST developer if `nest-simulator` is able to run in both architectures.

Reply:
They told me that it should work with `aarch64`. I removed `flathub.json`.